src/csv/report.rs

```diff
@@ -426,17 +434,24 @@ fn nominal_plot(table: &[HashMap<String, String>], column: String) -> Vec<PlotRe
     }
 
     let mut plot_data = Vec::new();
-    for (k, v) in count_values {
-        let plot_record = PlotRecord { key: k, value: v };
+    for (k, v) in &count_values {
+        let plot_record = PlotRecord {
+            key: k.to_owned(),
+            value: *v,
+        };
         plot_data.push(plot_record);
     }
 
     if plot_data.len() > 10 {
+        let unique_values = count_values.iter().map(|(_, v)| v).collect::<HashSet<_>>();
+        if unique_values.len() <= 1 {
```

Comment:
This should check for is_empty()

Reply:
`count_values` contains every unique entry from one column of the raw data as the key of the `HashMap` and how often the entry occurs as the value. If the resulting `HashSet` (that is taking the occurrences `v` from each entry) contains one element, each entry must occur equally often. An empty set would mean that there wouldn't be any values or am i getting something completely wrong?